src/course-unit/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -22,9 +22,20 @@ const Breadcrumbs = ({ courseId, parentUnitId }: { courseId: string, parentUnitI
     ? adoptCourseSectionUrl({ url, courseId, parentUnitId })
     : `${getConfig().STUDIO_BASE_URL}${url}`);
 
-  const getPathToCoursePage = (isOutlinePage, url) => (
-    isOutlinePage ? getPathToCourseOutlinePage(url) : getPathToCourseUnitPage(url)
-  );
+  // based on the level of breadcrumbs the url will differ
+  // at the subsection level it should navigate to the first unit if available
+  // if no unit then navigate to the subsection outline
+  function getPathToCoursePage(index, url, usageKey: string) {
+    let navUrl: string;
+    if (index === 0) {
+      navUrl = getPathToCourseOutlinePage(url);
+    } else if (index === 1) {
+      navUrl = subsectionFirstUnitEditUrl({ courseId, subsectionId: usageKey });
+    } else {
+      navUrl = getPathToCourseUnitPage(url);
```

Comment:
Eventually these other functions should be changed to use the `usageKey` too. Doesn't need to happen now though.